sdk/communication/azure-communication-chat/README.md

```diff
@@ -0,0 +1,714 @@
+    .setChatMessageId(messageId);
+
+// The unique ID of the thread.
+final String threadId = "<thread_id>";
+client.sendChatReadReceipt(threadId, readReceipt, new Callback<Void>() {
+    @Override
+    public void onSuccess(Void result, Response response) {
+        //TODO: Take further action
+    }
+
+    @Override
+    public void onFailure(Throwable throwable, Response response) {
+        // TODO: Display error message.
+    }
+});
+```
+
+#### Get read receipts
+
+Use the `listChatReadReceipts` method to retrieve read receipts for a thread.
+
+```java
+// The unique ID of the thread.
+final String threadId = "<thread_id>";
+client.listChatReadReceiptsPages(threadId,
+    new Callback<AsyncPagedDataCollection<ReadReceipt, Page<ReadReceipt>>>() {
+    @Override
+    public void onSuccess(AsyncPagedDataCollection<ReadReceipt, Page<ReadReceipt>> result,
+        Response response) {
+         // pageCollection enables enumerating list of chat members.
+         pageCollection.getFirstPage(new Callback<Page<ReadReceipt>>() {
+            @Override
+            public void onSuccess(Page<ReadReceipt> firstPage, Response response) {
+                for (ReadReceipt receipt : firstPage.getItems()) {
+                    //TODO: Take further action
+                }
+                retrieveNextReceiptsPages(firstPage.getPageId(), pageCollection);
+            }
+
+            @Override
+            public void onFailure(Throwable throwable, Response response) {
+                // TODO: Display error message.
+            }
+         }
+    }
+
+    @Override
+    public void onFailure(Throwable throwable, Response response) {
+        // TODO: Display error message.
+    }
+});
+
+void retrieveNextReceiptsPages(String nextPageId,
+    AsyncPagedDataCollection<ReadReceipt, Page<ReadReceipt>> pageCollection) {
+    pageCollection.getPage(nextPageId, new Callback<Page<ReadReceipt>>() {
+        @Override
+        public void onSuccess(Page<ReadReceipt> nextPage, Response response) {
+            for (ReadReceipt receipt : nextPage.getItems()) {
+                //TODO: Take further action
+            }
+            if (nextPage.getPageId() != null) {
+                retrieveNextReceiptsPages(nextPage.getPageId(), pageCollection);
+            }
+        }
+
+        @Override
+        public void onFailure(Throwable throwable, Response response) {
+            // TODO: Display error message.
+        }
+    });
+}
+```
+
+# Troubleshooting
+
+## General
+
+When an error occurs, the client calls the callback's `onFailure` method. You can use the provided `Throwable` to act upon the failure.
+
+```java
+client.createChatThread(thread, new Callback<MultiStatusResponse>() {
+    public void onFailure(Throwable throwable, okhttp3.Response response) {
+        // TODO: Display error message.
+    }
+});
+```
+
+# Next steps
+
+More sample code should go here, along with links out to the appropriate example tests.
+
+# Contributing
```

Comment:
We should agree to a standard about if we will have one empty line after any header or not, just to be consistent. Not required for this release.